default.yml

```diff
@@ -162,7 +166,7 @@ Lint/EmptyInPattern:
 Lint/UnexpectedBlockArity:
   Enabled: false
 
-# These code patterns are rare/nonexistent in our code, so checking for them is a waste of time
+# These code patterns are rare/nonexistent in our code or harmless, so checking for them is unnecessary.
```

Comment:
Is the performance gain so considerable that we explicitly disable these valid ones? `IncompatibleIoSelectWithFiberScheduler` could prove useful in hopefully not too distant future.